fxdc/config.py

```
@@ -157,5 +340,78 @@ def get_class_name(self, class_: type) -> str:
                 return customclass.classname
         return class_.__name__
 
+    def get_class(self, classname: str) -> Optional[_customclass]:
+        for customclass in self.custom_classes:
+            if customclass.classname == classname:
+                return customclass
+        return None
+
+    def export_config(self, file: str = "config.fxdc"):
```

Ruff (0.12.2)

Missing return type annotation for private function `export_config`

Add return type annotation: `None`

(ANN202)